### ISSUE TITLE
solrpy Issues an issue here with the repository.

### ISSUE THREAD

**Issue by search5** [lexpar/advent2017#1]
Hi Alex, I am going to take over the original solrpy lately and I found the contact information you wanted to drop solrpy3 that you registered with pypi. But I can not find the e-mail address, so I leave this post on another project you run.

I'd like you to drop the solrpy3 registered on pypi. If possible, I would like to download the solrpy3 package and just send a mail to search5@gmail.com.

**Comment by agpar** [comment]
Hi,

You want me to delete solrpy3 from pypy and send you a copy of the
last release?

I noticed the update to python3 for the main solrpy package is not yet
merged on GH.

Alex

**Comment by search5** [comment]
hi,

I added Python 3 support to solrpy. The original author, Ed, also received the pypi maintainer privilege.

The repository will be taken over soon, so I hope you remove the packages from pypi as much as possible.

**Comment by search5** [comment]
@lexpar We have updated the solrpy library that solved your problem with the answer you provided. The new update forced the response document to be requested in XML and was modified to work with both Python 2 and 3.

The solrpy project was acquired from the original author and modified. Now delete the newly added project and remove solrpy3 from pypi. The project I have acquired is github.com/search5/solrpy.

Thank you.
